.travis.settings.xml

```diff
@@ -1,3 +1,20 @@
+<!--
+  ~ Copyright 2020 The CloudEvents Authors
```

Comment:
Consider using `2018-Present` (legally valid form) to avoid having to bump it every year

Reply:
Mh i felt the same, let's see what @duglin thinks about it

Reply:
Also the same should apply here i suppose https://github.com/cloudevents/sdk-java/pull/140/files#diff-9879d6db96fd29134fc802214163b95aR189

Reply:
context: Spring projects have recently migrated to `XXXX-Present` scheme after a thoughtful review by the legal team to avoid having to deal with the year bumps that create a ton of noise in work in projects every year.

Reply:
> Also the same should apply here i suppose

I'd say... everywhere :) Bumps are PITA and we have a chance to avoid it early :)

Reply:
if some legal person someplace says it's "ok", then it sounds good to me :-)

Reply:
I should have probably mentioned that I am not a layer (but got advised by them :D)

https://github.com/reactor/reactor-core/blob/a1dab8ad71605bfcf5e0d185ebe6e0f9f5ef560d/buildSrc/build.gradle#L2

Reply:
@duglin i thought you're the legal guy here :smile:

Reply:
I think I should be insulted by that LOL